model-optimizer/extensions/middle/L2NormToNorm.py

```diff
@@ -87,13 +87,13 @@ def replace_pattern(self, graph: Graph, match: dict):
         normalizel2_name = output_name + '/normalizel2'
         rename_node(match['l2_normalize'], normalizel2_name)
 
-        normalize_node = NormalizeOp(graph, {'name': output_name, 'eps': y,
-                                             'across_spatial': 0, 'channel_shared': 0}).create_node()
+        normalize_node = create_op_node_with_second_input(graph, NormalizeOp,
+                                                          np.ones(shape=int64_array([match['input'].shape[-1]]),
```

Comment:
@jane-intel, non-reshape-able transformation.

Reply:
@lazarevevgeny do i have to fix these non-reshapable transformation by myself?

Reply:
No, I just want to point this issue to @jane-intel .

Reply:
Ok